config/environment.js

```diff
@@ -2,5 +2,7 @@
 'use strict'
 
 module.exports = function (/* environment, appConfig */) {
-  return { }
+  return {
+    modulePrefix: 'ember-dagre'
```

Comment:
I _think_ I remember something about this but can you jog my memory what this is doing?

Reply:
This is for the resolver - so that anything using this can import from ember-dagre. This was probably not necessary here. The imports for the tests confused me, and I forgot to roll this back.